examples/dataprofiler_expectations/ge_expect_column_value_confidence_for_data_label.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
I think this is looking good. I think there needs to be more of a story / example / use case built into this. Build a hypothetical business that is under regulatory oversight to ensure sensitive data much remain in a certain environment... over 50% confidence, for example, must be flagged when the model is that confident. Or in an ETL pipeline, the model should never process sensitive data, so in the ETL pipeline, we must validate that any data the that exists in the pipeline is 80% or lower confidence... etc...  CC @JGSweets, thoughts?

Reply:
This gets sticky bc if the model is unsure between 3 diff labels, it possible it is .35% confident between similar labels which overlap. very model dependent.